types/node/util.d.ts

```diff
@@ -1499,11 +1499,17 @@ declare module "util" {
      * @return The parsed command line arguments:
      */
     export function parseArgs<T extends ParseArgsConfig>(config?: T): ParsedResults<T>;
-    interface ParseArgsOptionConfig {
+
+    /**
+     * Type of argument used in {@link parseArgs}.
+     */
+    export type ParseArgsOptionsType = "boolean" | "string";
+
+    export interface ParseArgsOptionConfig {
```

Comment:
Would it be better to take the opportunity to disambiguate the names of these interfaces if they're going to be exported?

```suggestion
    export interface ParseArgsOptionDescriptor {
```
perhaps?

Reply:
I like it!